src/DynamoRevit/DynamoRevit.cs

```diff
@@ -449,6 +478,50 @@ private void LoadDynamoView()
             splashScreen.DynamicSplashScreenReady -= LoadDynamoView;
         }
 
+        // Start main Dynamo directly without splash screen.
+        private void LoadDynamoWithoutSplashScreen()
```

Comment:
I guess this function simply reflects logic before we enable the splash screen in D4R right?

Reply:
yes